src/Microsoft.SqlTools.ServiceLayer/TaskServices/SqlTaskOpration.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.SqlTools.ServiceLayer.TaskServices
+{
+    public class SqlTaskOpration
```

Comment:
actually this class is not needed anymore. I'll remove it

---
In reply to: [134044192](https://github.com/Microsoft/sqltoolsservice/pull/438#discussion_r134044192) [](ancestors = 134044192)